cms/templates/cms/home_page.html

```diff
@@ -68,45 +74,150 @@
           <h1>
-            MITx MicroMasters Programs: Bringing MIT to you
+            MITx MicroMasters Programs: Master your future
           </h1>
-          <h2>
-            Advance your career. Accelerate your master’s degree.
-          </h2>
+          <a id="actionButton" class="btn btn-primary text-uppercase px-5 py-2 action-button" data-fancybox href="https://www.youtube.com/embed/7lVvOSJpFGY?autoplay=true">Watch Now</a>
+          <p class="video-container"></p>
         </div>
       </div>
 
-      <div class="row sub-banner">
-        <div class="col-md-12 ">
+      <div class="row sub-banner current-programs-section">
+        <div class="col-md-12">
           <section class="sub-banner-section">
             <div class="sub-banner-text">
-              <h3>What is the MicroMasters<span class="trademark">&reg;</span> Program Credential?</h3>
-            <div class="youtubevideowrap">
-              <div class="video-container">
-                <iframe width="560" height="315" src="https://www.youtube.com/embed/7lVvOSJpFGY" frameborder="0" allowfullscreen></iframe>
-              </div>
-            </div>
-              <p>A MicroMasters program credential is a professional and academic credential for online learners from
-                anywhere in the world. To earn the credential, learners must pass an integrated set of MIT graduate-level courses on edX.org,
-                and one or more proctored exams. This credential is valuable in and of itself.  Additionally,
-                credential-holders can then apply for an accelerated, on campus, master’s degree program at MIT
-                or other universities.
-                {% if benefits_url %}
-                  All MicroMasters program credential holders are also considered
-                  <a href="{{benefits_url}}">affiliates of the MIT Alumni Association</a>.
-                {% endif %}
+              <h3>MITx MicroMasters<span class="trademark">&reg;</span> Programs</h3>
+            
+              <p>Advance your career or accelerate your Master’s degree with a graduate-level digital credential from MIT.</p>
+              <p>The MicroMasters program credential from MIT Open Learning is a professional and academic credential for online learners
+                 from anywhere in the world who seek focused, accelerated advancement. Enroll in a program—no admission required—and take 
+                 a series of graduate-level online courses, taught by MIT instructors, through edX.org. Earn a program credential by 
+                 completing the course and passing one or more proctored exams. Enjoy the benefits: Credential earners can also apply for 
+                 an accelerated master’s degree program at MIT and other pathway universities; and include your credential on professional 
+                 profiles; and more MicroMasters program credential earners also become <a href="{{benefits_url}}">affiliates of the MIT Alumni Association</a>.
```

Comment:
```suggestion
                 profiles. MicroMasters program credential earners also become <a href="{{benefits_url}}">affiliates of the MIT Alumni Association</a>.
```

Note that the latest doc I have omits the link. I'm trying to confirm if we want to keep it.